examples/intermediate/06_history_backup.py

```
@@ -0,0 +1,534 @@
+    print("-" * 50)
+    
+    # 도구 정의
+    def get_time():
+        """현재 시간 반환"""
+        return datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    
+    def calculate(expression: str):
+        """수식 계산"""
+        try:
+            return str(eval(expression))
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)